.github/workflows/build-rc-app.yml

```diff
@@ -71,6 +71,16 @@ jobs:
           KEYPWD_COMPAT: ${{ secrets.SIGNING_KEY_PASSWORD_INTERNAL_RELEASE }}
           KEYSTOREPWD_COMPAT: ${{ secrets.SIGNING_STORE_PASSWORD_INTERNAL_RELEASE }}
           ENABLE_SIGNING: ${{ secrets.ENABLE_SIGNING }}
+      - name: Create service_account.json
+        run: echo '${{ secrets.SERVICE_ACCOUNT_JSON }}' > service_account.json
+      - name: Deploy to alpha track
+        uses: r0adkll/upload-google-play@v1
+        with:
+          serviceAccountJson: service_account.json
+          packageName: com.wire.internal
```

Comment:
Can't we just throw away the green app?
It's useless, isn't it?

I think absolutely none is using it and it's just one extra bloat in our configs.

Reply:
Yes not used, but my idea is to use it to get playtest session build instead of using Google Drive

Reply:
The problem with playtest, I guess, is that some times we do it from one branch, some times from another...

Not sure how to configure that right now.

Reply:
Removing it then